### PR TITLE
Jetty 12.1.x jetty ee web xml

### PR DESCRIPTION
**Pull request by janbartel**
Check for a `WEB-INF/jetty-eeX-web.xml` before checking for `WEB-INF/jetty-web.xml`.

### CODE REVIEW

File: jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-ee10-web-xml/WEB-INF/jetty-ee10-web.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!-- // which is available at https://www.apache.org/licenses/LICENSE-2.0. -->
+<!-- // -->
+<!-- // SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 -->
+<!-- // ======================================================================== -->
+<!-- // -->
+
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://eclipse.dev/jetty/configure_10_0.dtd">
+<Configure class="org.eclipse.jetty.ee10.webapp.WebAppContext">
+  <Set name="contextPath">/raspberry</Set>
+</Configure>
```
Comment (gregw) [inline jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-ee10-web-xml/WEB-INF/jetty-ee10-web.xml]: All these files have bad end-of-line termination.

File: jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-ee10-web-xml/WEB-INF/jetty-web.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<!-- // which is available at https://www.apache.org/licenses/LICENSE-2.0. -->
+<!-- // -->
+<!-- // SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 -->
+<!-- // ======================================================================== -->
+<!-- // -->
+
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://eclipse.dev/jetty/configure_10_0.dtd">
+<Configure class="org.eclipse.jetty.ee10.webapp.WebAppContext">
+  <Set name="contextPath">/apple</Set>
+</Configure>
```
Comment (gregw) [inline jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-ee10-web-xml/WEB-INF/jetty-web.xml]: EOLN

File: jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-web-xml/WEB-INF/jetty-web.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<!-- // which is available at https://www.apache.org/licenses/LICENSE-2.0. -->
+<!-- // -->
+<!-- // SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 -->
+<!-- // ======================================================================== -->
+<!-- // -->
+
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://eclipse.dev/jetty/configure_10_0.dtd">
+<Configure class="org.eclipse.jetty.ee10.webapp.WebAppContext">
+  <Set name="contextPath">/orange</Set>
+</Configure>
```
Comment (gregw) [inline jetty-ee10/jetty-ee10-webapp/src/test/resources/webapp-with-jetty-web-xml/WEB-INF/jetty-web.xml]: EOLN

**Review by gregw — APPROVED** [review]
I couple of niggles.

Also, I think we need to ponder what to do with jetty-web.xml files?  Warn if we find them?  Perhaps warn if there is another jetty-eeX-web.xml that doesn't match?

Or maybe we make a `EeContextHandler` class that sits between the EEX impls and ContextHandler and warn only if a jetty-web.xml contains anything but the configuration for that common subclass.

Eitherway, this is good enough for us to commit to 12.1.x and get on with updating the demos to use it.

**Review by firoj0 — COMMENTED** [review]
EeContextHandler

**Pull request merged**